core/src/main/res/values/strings.xml

```diff
@@ -454,8 +454,11 @@
     <string name="pref_showDownloadReport_title">Show Download Report</string>
     <string name="pref_showDownloadReport_sum">If downloads fail, generate a report that shows the details of the failure.</string>
     <string name="pref_expand_notify_unsupport_toast">Android versions before 4.1 do not support expanded notifications.</string>
-    <string name="pref_queueAddToFront_sum">Add new episodes to the front of the queue.</string>
-    <string name="pref_queueAddToFront_title">Enqueue at Front</string>
+    <string name="pref_enqueue_location_title">Enqueue Location</string>
+    <string name="pref_enqueue_location_sum">Add episodes to: %1$s.</string>
+    <string name="enqueue_location_back">back of the queue</string>
+    <string name="enqueue_location_front">front of the queue</string>
+    <string name="enqueue_location_after_current">after current episode</string>
```

Comment:
Please use capital letters for options items

Reply:
Do you mean (e.g., capital b in Back to the queue)?
```
    <string name="enqueue_location_back">Back of the queue</string>
```

If so, the 1-line summary of the item would be like "Add episodes to: Back of the queue." We could still dynamically toLowercase it when adding it to the 1-line summary. It probably would work for all relevant language.

Reply:
That's what I mean, yes. In German (biggest user base), the translation for the word "queue" needs a capital letter. I think a capital letter after ":" is better there.

Now that I think about it, the translation of "Back of the queue" is quite long ("Am Ende der Abspielliste"). What about just using Back/Front/After current episode?

Reply:
done

Reply:
Done but not pushed?

Reply:
pushed